test/synth/synth-stdout.test.ts

```diff
@@ -413,3 +1114,47 @@ function requireSynth() {
   // eslint-disable-next-line
   return require(module);
 }
+
+async function expectSynthMatchSnapshot(workdir: string) {
+  const omitted = '__omitted__';
+
+  const files = await promisify(glob)('**', {
+    cwd: workdir,
+    nodir: true,
+  });
+
+  const map: Record<string, string> = {};
+
+  for (const file of files) {
+    const filePath = path.join(workdir, file);
+
+    if (file === CHART_YAML) {
+      const source = fs.readFileSync(filePath, 'utf-8');
+      const yamlContents = yaml.parse(source);
+
+      expect(yamlContents.name).toMatch(/cdk8s-/);
+      expect(yamlContents.description).toMatch(/Generated chart for cdk8s-/);
+
+      // These values are not stable
```

Comment:
How so? because its derived from the working directory? If that's the case, lets create a stable directory name inside a temporary directory.

Reply:
The dir now created for test would have consistent name. So this won't be an issue.